Update Home Depot

lazytownfan opened the pull request:
* Add call & e-mail as 2FA methods for Home Depot
* Discovered this after experimenting with new account

lazytownfan committed:
Update 2FA methods for Home Depot

Add call & e-mail as 2FA methods for Home Depot

hkamran80 commented on entries/h/homedepot.com.json:
```diff
@@ -3,7 +3,9 @@
     "domain": "homedepot.com",
     "url": "https://www.homedepot.com",
```
```suggestion
```

lazytownfan replied:
* Done (in another commit - I should've just done it here)
* I didn't create the Home Depot entry, just using what was already here

hkamran80 replied:
Thanks!

hkamran80 commented on entries/h/homedepot.com.json:
```diff
@@ -3,7 +3,9 @@
-      "sms"
+      "sms",
+      "call",
+      "email"
```
Can you provide a screenshot of the activation page? Please blur any personal information.

lazytownfan replied:
Here's a screenshot:

![upload](https://github.com/2factorauth/twofactorauth/assets/93500165/f98c986f-e1de-48d0-9ef4-77ade9cf4313)

(Fun fact: Signal Desktop has some more marker thickness controls for editing photos than Signal Android; not sure how Signal iOS is like.)

Also, I used manual redaction, as the [warning](https://www.privacyguides.org/en/data-redaction/#privacyblur) for PrivacyBlur on Privacy Guides indicates that it's possible to recover text that has been blurred.

So, drawing opaque rectangles (as [implied](https://nitter.1d4.us/Snowden/status/1423018727361060865#m) by Snowden on Twitter) or opaque redaction (such as the old-school way of redacting paper copies of classified documents) is the safer way of redacting text.

hkamran80 replied:
Is there another page that shows that phone calls and emails act as two-factor authentication?

lazytownfan replied:
So I had to turn off 2FA in order to see this again, which Home Depot calls "logging in without a password".  It's sort of laconic and I don't think Home Depot has any support page explaining what's going on.

This is what the page looks like before turning on Home Depot's 2FA, which is called "Sign In with a Password":
![upload1](https://github.com/2factorauth/twofactorauth/assets/93500165/981e5b3a-2ddd-44a5-ab6c-71283aaf0ff1)

When you select to activate 2FA, Home Depot calls this "Sign In without a Password":
![upload2](https://github.com/2factorauth/twofactorauth/assets/93500165/91f8b2d3-eeee-43c8-b406-2d8e2a13bfd3)

Although Home Depot requires a cell phone number to create an account, which will be verified with a code in an SMS message (or a SMS capable VOIP number); Home Depot will default to sending the 2FA code for initial setup and all subsequent logins via e-mail:
![upload3](https://github.com/2factorauth/twofactorauth/assets/93500165/946ea6be-da11-4626-b8ba-af4292cc9847)

This is what the page will look like upon successful activation of 2FA:
![upload4](https://github.com/2factorauth/twofactorauth/assets/93500165/554b5b55-25ca-4df2-9183-841bcd02f13b)

hkamran80 replied:
What their sign in without password describes is not 2FA.
> Instead of entering a password you may sign into your account using a one-time passcode that we will send to you.

What happens if you toggle the 2FA switch?

lazytownfan replied:
This is the login screen after turning on Home Depot's 2FA switch:
![upload cleaned](https://github.com/2factorauth/twofactorauth/assets/93500165/307b39a0-1230-457b-957f-b4ddbcbac51f)

You only need to enter the username (i.e., your e-mail) and then you receive the code from Home Depot.  Usually the site defaults to e-mail, but you can manually choose a SMS text or automated voice call to your phone number.

The site does not request for your password when using this method.

Carlgo11 replied:
@lazytownfan if a password isn't required then it's still just 1 factor.

lazytownfan replied:
So, would this not fall under 2FA Directory's definition of 2FA?

Carlgo11 replied:
Exactly

lazytownfan committed:
Delete obvious URL in Home Depot

Removing URL, as it is somewhat obvious, and as per the commit suggestion in PR #7593

Carlgo11 committed:
Switch Home Depot 2FA status

Switch from listed 2FA methods to contact methods as it seems clear they use 2SV and not 2FA.
